SU-HSDO/suhumsci
# Shs 77 megamenu unpublished styles

## Summary
This adds styles to visually differentiate unpublished nodes in the the mega menu.

## Steps to Test
1. Enable the new megamenu, go to the main menu (`/admin/structure/menu/manage/main`) and unpublish a page. 
2. Verify that it gets the golden background to signify its unpublished like in the screenshots below.

## Screenshots
![Screen Shot 2022-08-22 at 3 44 31 PM](https://user-images.githubusercontent.com/8405274/186006030-691c4cf9-c234-400a-a846-8b4003b1b172.png)

![Screen Shot 2022-08-22 at 3 44 25 PM](https://user-images.githubusercontent.com/8405274/186006039-b522b961-13a9-41c9-932a-e3fef4b6601f.png)

![Screen Shot 2022-08-22 at 3 44 45 PM](https://user-images.githubusercontent.com/8405274/186006060-08405c97-6007-403c-925a-a3ac28944cde.png)

## Notes
We'll add this after the current release to give enough time for testing.
